src/components/calculator.js

```diff
@@ -0,0 +1,42 @@
+const button = require("./button");
+const buttonsGroup = require("./buttons-group");
+
+function calculator(id) {
+	const buttonsTop = buttonsGroup("buttons-top", [
+		button("C","clear-all upper left"),
+		button("+/-","less-more upper"),
```

Comment:
i also changed this little things that was wrong, the button's positions and because i changed the components i had to reload the snapshot.
the most edited file was just the **main.css**

Reply:
I don't get why do you still need those classes/id

`clear-all upper left` and `less-more upper`